### PR TITLE
CoVID-19 Cases using Web Scraping

### LINKED ISSUE #1165: feature for COVID cases using Web Scraping
# Description

I want to add a python script for COVID-19 cases to check the number of cases in any country. I will be using web scraping concept to fetch the data from a website.

| name            | problem/motivation                                                  | possible solution  | assignees                                                      |
| --------------- | ------------------------------------------------------ | ------- | -------------------------------------------------------------- |
| COVID cases using Web Scraping | to check the number of cases in any country | The user will enter the country name and the script will fetch the data from a website using the concept of Web scraping to find the no. of cases, deaths, etc. | self-assign |

### PR DESCRIPTION
**Pull request by Umesh-01**
# Description
I have added a feature for checking Covid19 cases with a python script using the concept of web scraping, also added appropriate screenshots and a README.md file to explain the details for the module used in Web scraping.

This Fixes #1165 

## Type of change
- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works

### CODE REVIEW

File: Python/Covid_Cases_Scraper/README.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+```
+OR
+```python
+from bs4 import BeautifulSoup
+import requests
+url = "https://something.com/something"
+res = requests.get(url)
+data = BeautifulSoup(res.text, 'lxml')
+print(data)
+```
```
Comment (HarshCasper) [inline Python/Covid_Cases_Scraper/README.md:46]: Remove this

File: Python/Covid_Cases_Scraper/README.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+Beautiful Soup is a library that makes it easy to scrape information from web pages. It sits atop an HTML or XML parser, providing Pythonic idioms for iterating, searching, and modifying the parse tree.
+
+### Requests 
+
+Requests allows you to send HTTP/1.1 requests extremely easily. There’s no need to manually add query strings to your URLs, or to form-encode your PUT & POST data — but nowadays, just use the json method!
+
+## Installation
+
+Install with `pip` command in any terminal
```
Comment (HarshCasper) [inline Python/Covid_Cases_Scraper/README.md:17]: Provide instructions to use `requirements.txt` as well

File: Python/Covid_Cases_Scraper/README.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+To do this we need two libraries `beautifulsoup4` and `requests`
+
+### Beautifulsoup4
+
+Beautiful Soup is a library that makes it easy to scrape information from web pages. It sits atop an HTML or XML parser, providing Pythonic idioms for iterating, searching, and modifying the parse tree.
+
+### Requests 
+
+Requests allows you to send HTTP/1.1 requests extremely easily. There’s no need to manually add query strings to your URLs, or to form-encode your PUT & POST data — but nowadays, just use the json method!
```
Comment (HarshCasper) [inline Python/Covid_Cases_Scraper/README.md:13]: Remove this

**Review by HarshCasper — CHANGES_REQUESTED** [review]

File: Python/Covid_Cases_Scraper/README.md (hunk 1 above)
Comment (Umesh-01) [reply Python/Covid_Cases_Scraper/README.md:46]: done

File: Python/Covid_Cases_Scraper/README.md (hunk 3 above)
Comment (Umesh-01) [reply Python/Covid_Cases_Scraper/README.md:13]: done

File: Python/Covid_Cases_Scraper/README.md (hunk 2 above)
Comment (Umesh-01) [reply Python/Covid_Cases_Scraper/README.md:17]: done

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**